cansat-gps/tests/main.rs

```diff
@@ -15,29 +16,38 @@ fn gps_last_nmea_returns_none_if_no_msg_received() {
     assert!(gps.last_nmea().is_none());
 }
 
-#[ignore]
 #[test]
 fn gps_last_nmea_returns_msg_if_msg_received() {
     let uart = mock::Serial::new(FIRST_MSG.to_owned());
-    let _ = Gps::new(uart);
+    let mut gps = Gps::new(uart);
+
+    while let (_, _is_msg_terminated @ false) = gps.read_serial().unwrap() {}
 
-    todo!()
+    assert_eq!(gps.last_nmea().unwrap(), FIRST_MSG);
 }
 
-#[ignore]
 #[test]
 fn gps_last_nmea_returns_first_msg_if_second_not_yet_terminated() {
     let uart = mock::Serial::new([FIRST_MSG, SECOND_MSG].concat());
-    let _ = Gps::new(uart);
+    let mut gps = Gps::new(uart);
+    let mut rng = rand::thread_rng();
+
+    while let Ok((_read_data, false)) = gps.read_serial() {}
```

Comment:
last change and it should be good to merge
```suggestion
    while let (_, _is_msg_terminated @ false) = gps.read_serial().unwrap() {}
```